src/main/java/jenkins/scm/impl/SingleSCMNavigator.java

```diff
@@ -72,7 +72,12 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
 
         @Override
         public String getDisplayName() {
-            return "Single Repository";
+            return Messages.SingleSCMSource_DisplayName();
+        }
+
+        @Override
+        public String getDescription() {
+            return Messages.SingleSCMSource_Description();
```

Comment:
:ant: pointless to override this, since it returns null from `newInstance` so this method will never be called.